airflow/example_dags/tutorial.py

```diff
@@ -95,7 +98,7 @@
     depends_on_past=False,
     bash_command=templated_command,
     params={'my_param': 'Parameter I passed in'},
-    dag=dag)
+    dag=dag,
+)
 
-t2.set_upstream(t1)
-t3.set_upstream(t1)
+t1 >> [t2, t3]
```

Comment:
And this?

Reply:
Same here, but using [__rshift__()](https://github.com/apache/incubator-airflow/blob/39a9365b6250e588363e59dd6e332e02a28f411d/airflow/models.py#L2643-L2656).